scripts/issue/441/mint_lowest_range_uni_v3.py

```diff
@@ -0,0 +1,50 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+from brownie import Contract, interface
+
+wbtc = Contract(registry.eth.treasury_tokens.WBTC)
+
+RANGE_0 = 0.0000313
+RANGE_1 = 0.000063
```

Comment:
think this should be `.0000625`, but in the previous script we also deployed from `.000063` so better to go with that now.

https://github.com/Badger-Finance/badger-multisig/blob/a5e97213ac0746a7d32fa36fb848c8705ccd53db/scripts/issue/32/mint_lowest_range_uni_v3.py#L7